src/containers/RepositoriesContainer/RepositoriesDucks.js

```diff
@@ -0,0 +1,37 @@
+// Actions
+const LOAD = 'react-redux-starter-app/repositories/LOAD';
+const LOAD_SUCCESS = 'react-redux-starter-app/repositories/LOAD_SUCCESS';
+const LOAD_FAIL = 'react-redux-starter-app/repositories/LOAD_FAIL';
```

Comment:
`react-redux-starter-app` then is redundant as you won't have actions from somewhere else, right? :)